stable/aws-events-exporter/values.yaml

```diff
@@ -0,0 +1,42 @@
+# Default values for aws-events-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicas: 1
+
+image:
+  repository: thomasnyambati/aws-events-exporter
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: 1.0.0
+
+imagePullSecrets: []
+nameOverride: ""
+fullnameOverride: ""
+# serviceAccountName -- service account to be used by the containers
+serviceAccountName: ""
+podAnnotations: {}
+
+podSecurityContext: {}
+
+securityContext: {}
+
+service:
+  type: ClusterIP
+
+resources: {}
+nodeSelector: {}
+tolerations: []
+affinity: {}
+
+exporter:
+  # exporter.port -- Exporter port
+  port: 9090
+  # exporter.listenAddress -- Exporter listening address
+  listenAddress: 0.0.0.0
+  # exporter.queueUrl -- SQS queue url
+  queueUrl: ""
```

Comment:
Is this optional? Because from the chart description it sounds like it's required? But setting `""` makes it optional?